Weather Widget

kamufozi opened the pull request.

andre-49 commented:
Better UI for the loading phase, have it display in the widget itself, the borders need better styling as well, one that matches the figma. check the colors I used in the clock or we can change them if they're off.

kamufozi commented:
> Better UI for the loading phase, have it display in the widget itself, the borders need better styling as well, one that matches the figma. check the colors I used in the clock or we can change them if they're off.

I added a spinner loading phase and adjusted my border style to look like yours
